packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1218,3 +1219,23 @@ Posts.addView("stickied", (terms: PostsViewTerms) => (
     },
   }
 ));
+
+
+Posts.addView("nominatablePostsByVote", (terms: PostsViewTerms) => {
+  return {
+    selector: {
+      _id: {$in: terms.postIds},
+      userId: {$ne: terms.userId},
```

Comment:
This is a security vulnerability that enables deanonymizing all votes through the API

Reply:
Wait how? (the pieces here don't actually intrinsically have anything to do with votes. Am I missing a larger interaction?)

Reply:
Oops, this one isn't actually a vuln (I was thinking this one was also on the votes collection like the other one).